src/renderer/renderer.h

```diff
@@ -0,0 +1,29 @@
+
+#ifndef __RENDERER_H__
+#define __RENDERER_H__
+
+#define _USE_MATH_DEFINES
```

Comment:
is this crossplat? I think this is windows only? <ore importantly since this is a c++ project we don't need a math.h vs <cmath> toggle. just add #include <cmath> and remove line 8''s math.h.

Reply:
This should have been removed before the commit. It was removed in one of my changes I just pushed.